index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
Please format this document using prettier and push the changes

Reply:
alright